backend/mcp_backend/routers/crew_mcp.py

```
@@ -0,0 +1,422 @@
+                # Run the title and tag generation in the background
+                import asyncio
+
+                # Create background task for title/tag generation
+                async def safe_background_task():
+                    try:
+                        await generate_title_and_tags_background(
+                            request_data, request, result, user
+                        )
+                    except Exception:
+                        # Completely isolate any background task exceptions
+                        pass
```

Ruff (0.12.2)

Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

(SIM105)